### DISCUSSION TITLE
[Kavita] Auto-add new publishers as libraries

### DISCUSSION THREAD

**Discussion by janaxhell** [Kareadita/Kavita#3064] [Ideas]
### Idea Description

I use Mylar, set to sort comics into (\mylar)\publisher\series\issues 123.cbx
The publishers of Mylar alone are currently 328 and now and then they grow. (I have other pre-Mylar libraries)
It would be VERY nice if Kavita could auto-add (new) subfolders (the \publishers) of a certain folder (i.e \mylar, or \Italy, etc) as Library on the left column menu as they are added by Mylar, or manually from other sources. 
I know I can access publishers via metadata, but IMO it's not as convenient as having a neat separate entry for each on the side. 

### Idea Category

User Experience

### Duration of Using Kavita

2 years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Cducharme84** [comment]
While I don't think this one's gonna get much overall traction as Kavita only adds new folders to existing libraries and doesn't add libraries without the admin initiating via UI/API there are some fan (and one team member with Diesel getting in on the action) scripted solutions for this that do just that and could be run on a timed job. If you're on the Kavita Discord check out the #share-your-own channel for things like that.

**Reply by janaxhell** [reply]
Ok, will do, thanks.
